website/blog/2020-06-30-data-lake-analytics-blog.md

```diff
@@ -0,0 +1,132 @@
+<br><br>
+This layer emphasizes a very important feature in Cloud Native computing, that is, Serverless is server-free and is the basic service for building SaaS on the cloud, except for Data Lake analysis scenarios, there is also a more ubiquitous serverless PaaS and Function-computing as-a-Service (FaaS) such as Alibaba Cloud Function Calculation. This article focuses on the serverless analysis capability of Data Lake analytic scenarios.<br>
+
+1. Elasticity - This allows an on-demand flexibility capability, which means being flexible in time, predictable and intelligent mixed load handling ability<br>
+
+2. Resiliency - User’s unperceived High Availability capability includes transparent rolling upgrade, fast handover, disaster tolerance across AZ (available area), etc<br>
+
+3. Federated and Analytical - Multi-mode federal analysis capabilities, includes analysis and integration capabilities for multiple formats and systems, comprehensive analysis functions and excellent interactive analysis performance and experience, parallel computing processing capabilities, compatible interface capability and so on.<br>
+
+<br><br>
+
+**Third layer**: Data analysis application and visualization application layer
+<br><br>
+Business logic based upon Data Lake analysis service layer may require an integration of data analysis tools on the cloud. All kinds of visualized data analysis products and tools are available to be interacted with users.
+<br>
+Upon the basis of the above cloud data lake analysis architecture, the data federation ecosystem is divided into three layers accordingly, for which Data Lake Analytics plays a key role for serverless cloud-native computing framework:
+<br><br>
+
+![](/img/blog/2020-06-30-data-lake-analytics-blog/dla_three_tier.png)
+
+<br><br>
+
+
+
+## Data Lake Analytics
+### Introducing DLA 
+
+Data Lake Analytics, known as DLA, is a large scale serverless data federation service on Alibaba Cloud. It is one of the most popular serverless SQL engines based on a well-customized computing engine from PrestoDB. DLA integrates with mainstream data sources and provides easy-to-use MySQL JDBC/ODBC connection protocol to allow users to interact with.
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/DLA_architecture.png)
+<br><br>
+FrontNode is the SQL access layer which provides MySQL protocol to let users interact with. Since MySQL is a very flexible and compatible service, it becomes a user-friendly connection protocol to be used by DLA users. Unified Meta Layer is a metadata repository which holds ALL metadata for all data sources DLA can support. DLA was forked from PrestoDB Version 0.227 and is now integrated with mainstream data sources in Alibaba Cloud. It is currently working closely with the PrestoDB community to track ongoing projects and updates. It is mission critical to continue the close partnership with PrestoDB for business success with latest updates.
```

Comment:
> Since MySQL is a very flexible and compatible service, it becomes a user-friendly connection protocol to be used by DLA users.

This reads as MySQL becoming a connection protocol. I guess you mean sth like MySQL has a user friendly connection protocol. Please revise.

Reply:
"ALL metadata" => all metadata (don't need caps)

Reply:
"PrestoDB Version 0.227" => PrestoDB (v0.227)

Reply:
yah the sentense is confusing, revised